code/game/objects/items/tanks/tanks.dm

```diff
@@ -80,21 +84,31 @@
 
 	populate_gas()
 
+	reaction_info = new
+	explosion_info = new
```

Comment:
```suggestion
	reaction_info = list()
	explosion_info = list()
```
Your code works, but we tend to use `list()` to initialize them throughout the codebase.
Makes it a little bit more clear it's not some other kind of datum.